### DISCUSSION TITLE
Resume Checkpoint

### DISCUSSION THREAD

**Discussion by LMBits** [JoePenna/Dreambooth-Stable-Diffusion#62] [Q&A] [answered]
Is it possible to resume a checkpoint? So for instance I made a mistake and put 2020 steps but the whole epoch0 meant to be 11100 so it stops at 20%. What can I do from there? Start all over or can I resume somehow? Also, what would you do if you want to keep going with epoch1 if you already completed the previous epoch0?

**Comment by cryptoguard** [comment]
Digging this question from the dirt but, anyone figured that out yet?

**Reply by SAC020** [reply, marked as answer]
rename your (partially trained) checkpoint to model.ckpt, skip the stable diffusion download step, train as usual, profit
